generator/src/init-sources.ts

```diff
@@ -145,6 +151,11 @@ export class InitSources {
         // first, clone
         await this.clone(extension);
 
+        // update che-theia location
+        if (originalSource === 'https://github.com/eclipse/che-theia') {
+            this.cheTheiaLocation = extension.clonedDir;
```

Comment:
does it mean it won't work for forked repos?

Reply:
what do you suggest ? Check the che-theia name ?
Else I can merge definition for all repositories

Reply:
reworked